src/cockpit/channel.py

```diff
@@ -98,6 +98,7 @@ class Channel(Endpoint):
     payload: ClassVar[str]
     restrictions: ClassVar[Sequence[Tuple[str, object]]] = ()
 
+    group = 'default'
```

Comment:
This looks dangerous. I understand these as class variables, but it should be per-instance. It actually *seems* to work (with a little toy example), so maybe these are magically instantiated afresh for each new object instance?

But either way, I think this should go. The group name already gets assigned with a "default" default, so it's redundant.

Reply:
The way it works: each instance has a dictionary that contains all of its instance variables.  If you do a lookup of an instance variable, and it's not in the dictionary, it falls back to looking it up on the class.

I mostly assign the value I do here in order to make sure the typing is right.  I could also say `group: str` but then I should theoretically initialize it from `__init__()` instead of `do_open()`...

See also: channel is assigned a default value of `''` in the same way, and we do this with `= None` all over the place for variables that are meant as instance variables.

Reply:
Thanks for the explanation wrt. instance dictionary! This makes total sense. Fix LGTM!